lib/class-wp-rest-global-styles-controller-gutenberg.php

```diff
@@ -127,6 +149,23 @@ public function register_routes() {
 					'permission_callback' => array( $this, 'update_item_permissions_check' ),
 					'args'                => $this->get_endpoint_args_for_item_schema( WP_REST_Server::EDITABLE ),
 				),
+				array(
+					'methods'             => WP_REST_Server::DELETABLE,
+					'callback'            => array( $this, 'delete_item' ),
+					'permission_callback' => array( $this, 'delete_item_permissions_check' ),
+					'args'                => array(
+						'id'    => array(
+							'description'       => __( 'The id of a template', 'gutenberg' ),
```

Comment:
Did you mean the ID of a style variation? 

If yes, this is a pre-existing mistake in the corresponding `READABLE` endpoint above